Document Overseerr issue management features

AmGarera opened the pull request:
## Description

This PR documents the new issue management functionality for the Overseerr integration, including:
- 6 new issue sensors (total, open, closed, video, audio, subtitle)
- 4 new service actions (get_issues, create_issue, update_issue, delete_issue)
- Updated supported functionality section
- Example automations for issue management
- Use cases and workflow examples

## Related PRs

- Sensors implementation: https://github.com/home-assistant/core/pull/158888
- Services implementation: TBD (will be separate PR after sensors merge)
- python-overseerr enhancement: https://github.com/joostlek/python-overseerr/pull/291

## Notes

Documentation covers both the sensors (PR #158888) and services (upcoming PR) as they form a complete feature set. This PR should be merged after both implementation PRs are merged.

c0ffeeca7 commented:
Thank you for contributing, @AmGarera  :+1: 

1. It seems this PR was created based against the current branch. This is why there are unrelated commits and changes in the PR.
    <img width="882" height="115" alt="image" src="https://github.com/user-attachments/assets/3789fd0b-4502-43ed-b223-91ab889429a4" />

2. To resolve this, please [rebase](https://git-scm.com/docs/git-rebase) and point this PR against the `next` branch.
3. Restore the original PR template.
4. In the PR template, add the link to the related core PR. 
5. Add the link to the related brands PR.

AmGarera committed:
Document Overseerr issue management functionality

Add documentation for new issue management features:
- 6 new issue sensors (total, open, closed, video, audio, subtitle)
- 4 new issue services (get_issues, create_issue, update_issue, delete_issue)
- Example automations for automatic issue creation and monitoring
- Updated use cases section

c0ffeeca7 commented on source/_integrations/overseerr.markdown:
```diff
@@ -85,11 +107,59 @@ Get a list of media requests using `overseerr.get_requests`.
 | `sort_order`      | Yes      | The sort order to sort the results in (`added`/`modified`). |
 | `requested_by`    | Yes      | Filter the requests based on the user ID of the requester.  |
 
+### Action `overseerr.get_issues`
```
For the actions documentation, please apply the style that is suggested here:
https://github.com/home-assistant/home-assistant.io/blob/current/source/_integrations/_integration_docs_template.markdown?plain=1#L139

- Use a non-YAML title, mentioning the YAML in the introduction of the action description.
- Use a list instead of a table, as shown in the example.

AmGarera replied:
Should be all resolved now

AmGarera committed:
Use list format and non-YAML titles for action documentation

- Changed action section titles to non-YAML format (e.g., 'Get requests' instead of 'Action overseerr.get_requests')
- Mention the YAML action name in the introduction sentence
- Replaced tables with list format for better readability
- Format: **field_name** (*Required/Optional*): Description

c0ffeeca7 reviewed (dismissed):
Thank you, @AmGarera :+1:

joostlek commented on source/_integrations/overseerr.markdown:
```diff
@@ -68,28 +71,85 @@ There are sensors for:
+- `overseerr.get_issues` - Get a list of issues
+- `overseerr.create_issue` - Create a new issue for media
+- `overseerr.update_issue` - Update an existing issue
+- `overseerr.delete_issue` - Delete an issue
```
Not yet, let's make this PR only about the new sensors

AmGarera committed:
updating docs to only be about sensors

AmGarera committed:
fixed typo

c0ffeeca7 reviewed (approved):
Thank you, @AmGarera :+1:
